serniki/sernik-codilime.md

```diff
@@ -0,0 +1,80 @@
+Sernik nazwany na cześć firmy [CodiLime](https://www.codilime.com/), w której
+pracuję, i która finansuje badania nad tym przepisem (zjedli już kilkanaście
+sztuk ^^)
+
+## Składniki na dużą tortownicę (26-29 cm średnicy)
+
+- 800-1000 g sera (patrz komentarz poniżej)
+- 200-400 g śmietany lub śmietanki (patrz komentarz poniżej)
+- 5 dużych jajek (klasa wagowa 63-73 g)
+- 1/2 kostki Kasi lub masła
+- 1 szkl. drobnego cukru do wypieków (220g)
+- łyżeczka cukru waniliowego
+- 1 budyń waniliowy bez cukru (40g)
+- paczka okrągłych biszkoptów (ok. 100-120g, zależnie od rozmiaru tortownicy)
+- Opcjonalnie dodatek aromatu cytrynowego albo pomarańczowego
+
+*Uwaga: rodzaj użytego sera ma duże znaczenie. Najbardziej lubię używać twarogu
+zmielonego z Wielunia (ew. Piątnicy), który jest dość gęsty i słodki - wtedy
+żeby dodać mu charakteru i rozrzedzić go dodaję kwaśnej śmietany (w proporcji
+3:1). Jeśli Twój ser jest sam z siebie dość kwaśny, użyj słodkiej śmietanki, a
+proporcje ser-śmietana dobierz w zależności od gęstości sera.*
+
+*Oczywiście należy unikać słonych serów (takich jak mają w USA, np.
+Philadelphia). W 1kg sera powinno być nie więcej niż 600 mg sodu (=1500 mg
+soli).*
+
+*Ponieważ w USA ciężko dostać odpowiedni ser, czasem stosuję mieszankę quarka,
+whipped cream cheese i mascarpone (w proporcjach 2:1:1).*
+
+## Składniki na polewę
+
+- 50g czekolady mlecznej
+- 100g czekolady gorzkiej
+- 100g śmietanki 30% lub 36% (słodkiej)
+- wiórki kokosowe
+
+*Uwaga: śmietanka powinna mieć płynną konsystencję (nie używaj Piątnicy, która
+jest zagęszczana - polewa z użyciem Piątnicy nie rozprowadza się dobrze.
+Ewentualnie można rozrzedzić ją mlekiem).*
+
+## Procedura
+
+Wyjąć składniki z lodówki godzinę wcześniej.
+
+Dno tortownicy wyłożyć papierem do pieczenia (lub posmarować tłuszczem), ułożyć
+na dnie jedną warstwę biszkoptów.
+
+Oddzielić żółtka od białek, uważając aby do białek nie dostała się więcej niż
+kropla żółtka.
```

Comment:
Brzmi tak, jakby miała się dostać dokładnie jedna kropla, a chyba nie o to chodzi? :)